Misc/NEWS.d/next/Library/2020-09-28-04-56-04.bpo-14243.YECnxv.rst

```diff
@@ -0,0 +1,16 @@
+At the moment  ``NamedTemporaryFile`` is too hard to use portably when you

```

Comment:
This is way more details than we want here - this will end up in a list with 1000 other changes.

Most of this should move to specific documentation, and then this entry can link to it with ``:class:`NamedTemporaryFile` ``

Reply:
This is fixed. The spesific documentation is updated and linked to with ` :class:`tempfile.NamedTemporaryFile``